client/data/followers/use-followers-query.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * External dependencies
+ */
+import { useInfiniteQuery } from 'react-query';
+import { uniqueBy } from '@automattic/js-utils';
+
+/**
+ * Internal dependencies
+ */
+import wpcom from 'calypso/lib/wp';
+
+const MAX_FOLLOWERS = 10; // means pages (= 1000 followers);
+const defaults = {
+	max: 100,
+};
+
+/**
+ * Normalizes a follower object. Changes 'avatar' to 'avatar_URL' allowing a follower
+ * object to be used with the Gravatar component.
+ *
+ * @param  {object} follower A follower ojbect
+ * @returns {object}          A normalized follower object
+ */
+export function normalizeFollower( follower ) {
+	follower.avatar_URL = follower.avatar;
```

Comment:
This assignment should not mutate the `follower` object. We're mutating an object that is in React Query cache and the `avatar_URL` field then will be also on the supposedly raw `data.pages[ x ].subscribers[ y ]` response.

Reply:
Good catch @jsnajdr, will add a fix 👍🏻

Reply:
Would https://github.com/Automattic/wp-calypso/pull/50378/commits/14f119d270443f462355eab2d22e8d5f77ec7aec work as a fix?